brevo/assets/logs/brevo_tests.yaml

```diff
@@ -590,121 +590,134 @@ tests:
-        "X-Mailin-custom" : "some_custom_header",
+        "date" : "2024-10-03 11:29:38",
+        "reason" : "sent",
+        "subject" : "test",
+        "ts_event" : 1727935178,
         "sending_ip" : "10.10.10.10",
-        "subject" : "My first Transactional",
-        "message-id" : "2022283@abx.abc.com",
-        "template_id" : 22,
-        "id" : 11111,
+        "service" : "transactional-events",
+        "message-id" : "<20.21@s-ry.mn.fr>",
+        "template_id" : 27,
+        "id" : 1141551,
+        "tag" : "",
         "event" : "delivered",
-        "ts_event" : 1604933654,
-        "email" : "example@domain.com",
-        "ts" : 1604933619,
-        "tags" : [ "transac_messages" ]
+        "ts_epoch" : 1727935178000,
+        "email" : "abc@test.com",
+        "ts" : 1727935178,
+        "sender_email" : "abc@test.py"
       }
     service: "transactional-events"
     result:
       custom:
-        X-Mailin-custom: "some_custom_header"
-        date: "2020-10-09 00:00:00"
+        date: "2024-10-03 11:29:38"
         evt:
           name: "delivered"
-        id: 11111
-        message-id: "2022283@abx.abc.com"
+        id: 1141551
+        message-id: "<20.21@s-ry.mn.fr>"
         network:
           client:
             geoip: {}
             ip: "10.10.10.10"
+        reason: "sent"
+        sender_email: "abc@test.py"
+        service: "transactional-events"
         status: "Success"
-        subject: "My first Transactional"
-        tags:
-        - "transac_messages"
-        template_id: 22
-        ts: 1604933619
-        ts_event: 1604933654
+        subject: "test"
+        tag: ""
+        template_id: 27
+        timestamp: 1727935178000
+        ts: 1727935178
+        ts_event: 1727935178
         usr:
-          email: "example@domain.com"
+          email: "abc@test.com"
       message: |-
         {
-          "date" : "2020-10-09 00:00:00",
-          "X-Mailin-custom" : "some_custom_header",
+          "date" : "2024-10-03 11:29:38",
+          "reason" : "sent",
+          "subject" : "test",
+          "ts_event" : 1727935178,
           "sending_ip" : "10.10.10.10",
-          "subject" : "My first Transactional",
-          "message-id" : "2022283@abx.abc.com",
-          "template_id" : 22,
-          "id" : 11111,
+          "service" : "transactional-events",
+          "message-id" : "<20.21@s-ry.mn.fr>",
+          "template_id" : 27,
+          "id" : 1141551,
+          "tag" : "",
           "event" : "delivered",
-          "ts_event" : 1604933654,
-          "email" : "example@domain.com",
-          "ts" : 1604933619,
-          "tags" : [ "transac_messages" ]
+          "ts_epoch" : 1727935178000,
+          "email" : "abc@test.com",
+          "ts" : 1727935178,
+          "sender_email" : "abc@test.py"
         }
       service: "transactional-events"
       status: "ok"
       tags:
       - "source:LOGS_SOURCE"
-      timestamp: 1
+      timestamp: 1727935178000
   - sample: |-
```

Comment:
For example this one still does not have a timestamp property? Is it intended to to sometimes use `ts_epoch`?

Reply:
Yes, ts_epoch is used intentionally for transactional events. Both event types provide `ts_event (in seconds)`, but transactional events also include `ts_epoch (in milliseconds)`.

Marketing: `timestamp = ts_event * 1000`
Transactional: `timestamp = ts_epoch`
This timestamp is then used for the date remapper.